sourcemash/frontend/templates/item-card.tpl

```diff
@@ -17,7 +17,9 @@
       <div class="card-action valign-wrapper row">
         <div class="category-list flow-text col s8">
           <% for (var i = 0; i < item.categories.length; i++) { %>
-            <a href="/#categories/<%=item.categories.at(i).get('id')%>"><%=item.categories.at(i).get('name')%></a>
+            <% if (item.categories.at(i).get('item_count') > 1 || !user.get('email')) { %>
```

Comment:
When we have too many categories, the cards don't look that great and it won't mean much to a not-logged-in user anyway. Can we cap the number of categories at 5 or 6 for a not logged in user (aka add an if statement to break after ```i``` reaches a certain point?)?